src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,45 @@
+package seedu.addressbook.data.person;
+
+/**
+ * Represents a Person's common details in a contact field in the address book.
+ *
+ */
+public class Contact {
+
+    private String value;
+    private boolean isPrivate;
+
+    public Contact(String value, boolean isPrivate) {
+        this.value = value;
+        this.isPrivate = isPrivate;
+    }
+
+    @Override
+    public String toString() {
+        return value;
+    }
+
+    @Override
+    public int hashCode() {
```

Comment:
In general, `equals()` and `hashCode()` methods should be overridden together. Find out why!